rotkehlchen/api/v1/schemas.py

```diff
@@ -1743,8 +1743,11 @@ class ManualPriceSchema(Schema):
     price = PriceField(required=True)
 
 
-class TimedManualPriceSchema(ManualPriceSchema):
+class SingleTimestampSchema(ManualPriceSchema):
     timestamp = TimestampField(required=True)
+    from_asset = AssetField(required=False)
```

Comment:
What are you doing here? This is not what I told you to do: https://github.com/rotki/rotki/pull/4187#discussion_r850336293

I said to just rename it to `SingleTimestampSchema`, not to add 3 more fields. Why did you add those?

Reply:
ping. You have not answered this comment. What are you doing here?